uuid-ballerina/build.gradle

```diff
@@ -103,31 +106,56 @@ task revertTomlFile {
     }
 }
 
-task ballerinaBuild {
-    inputs.dir file(project.projectDir)
+task ballerinaTest {
+    dependsOn(":uuid-native:build")
+    dependsOn(copyStdlibs)
+    dependsOn(updateTomlVerions)
     finalizedBy(revertTomlFile)
 
-    doLast {
-        def distributionBinPath =  project.projectDir.absolutePath + "/build/target/extracted-distributions/jballerina-tools-zip/jballerina-tools-${ballerinaLangVersion}/bin"
-        def additionalBuildParams = ""
+    def debugParams = ""
+    if (project.hasProperty("debug")) {
+        debugParams = "--debug ${project.findProperty("debug")}"
+    }
 
-        if (project.hasProperty("skipBallerinaTests")) {
-            additionalBuildParams = "--skip-tests"
-        }
+    def balJavaDebugParam = ""
+    if (project.hasProperty("balJavaDebug")) {
+        balJavaDebugParam = "BAL_JAVA_DEBUG=${project.findProperty("balJavaDebug")}"
+    }
 
-        if (project.hasProperty("debugBallerina")) {
-            def port = project.getProperty("debugBallerina").toString();
-            additionalBuildParams += "--debug ${port}"
+    doLast {
+        exec {
+            workingDir project.projectDir
+            environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
+            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                commandLine 'cmd', '/c', "$balJavaDebugParam $distributionBinPath/ballerina.bat test ${debugParams} && exit %%ERRORLEVEL%%"
+            } else {
+                commandLine 'sh', '-c', "$balJavaDebugParam $distributionBinPath/ballerina test ${debugParams}"
+            }
         }
+    }
+}
+
+test {
+    dependsOn(ballerinaTest)
+}
+
+task ballerinaBuild {
+    inputs.dir file(project.projectDir)
+    dependsOn(":uuid-native:build")
+    dependsOn(copyStdlibs)
+    dependsOn(updateTomlVerions)
+    dependsOn(test)
+    finalizedBy(revertTomlFile)
 
+    doLast {
         // Build and populate caches
         exec {
             workingDir project.projectDir
             environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
             if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                commandLine 'cmd', '/c', "$distributionBinPath/ballerina.bat build ${additionalBuildParams} -c && exit %%ERRORLEVEL%%"
+                commandLine 'cmd', '/c', "$distributionBinPath/ballerina.bat build --skip-tests -c && exit %%ERRORLEVEL%%"
```

Comment:
Any reason to skip tests?

Reply:
The `ballerinaTest` task is added, which `ballerinaBuild` task depends on.

Reply:
Ack